### PR TITLE
chore(package): move dev dependencies

### PR DESCRIPTION
**Pull request by mt-roberto**
Moved devDependencies to dependencies

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,21 +20,20 @@
...
   "peerDependencies": {
-    "react": "^16.8.6"
-  },
-  "dependencies": {
-    "@svgr/core": "^4.3.0",
-    "ejs": "^2.6.2",
-    "fs-extra": "^8.0.1",
-    "svgo": "^1.2.2",
+    "react": "^16.8.6",
     "to-pascal-case": "^1.0.0"
```
Comment (mt-micky) [inline package.json]: So I need to put `to-pascal-case` as a dependency of my project to simply use icons?
I fail to see how this is a good thing

Comment (mt-roberto) [reply package.json]: You don't have to, that is automatically downloaded when you install `mt-web-icons`.

Comment (mt-micky) [reply package.json]: It’s not

File: package.json [hunk 2]
```diff
@@ -20,21 +20,20 @@
...
   },
-  "devDependencies": {
+  "dependencies": {
```
Comment (mt-micky) [inline package.json]: All of this should not be dependencies. It doesn’t make any sense to have to install all this to use a simple icon library

**Pull request merged**